main.tf

```diff
@@ -4,36 +4,36 @@
 provider "aws" {
   region  = "us-east-1"
   alias   = "virginia"
-  version = "1.50"
+  version = "~> 2.18"
```

Comment:
I think we can just go `>= 2.18` here, since we are not really requiring a specific provider version in our module, I think it's better to just always test with the latest? 🤷‍♂

Reply:
I doubt that >=2.18 would ever be an issue.  However I think that just like calling other modules it is good to lock the things used in the module to specific versions